### ISSUE TITLE
Allow manually increase the job timeout value

### ISSUE THREAD

**Issue by tuovinensanttu** [andersinno/kolga#19]
We have been running in to issue where application doesn't have enough time to start inside container before timeout terminates it. 

There is hard coded value [here](https://github.com/andersinno/kolga/blob/c0a68ef7ab584c2ca524aeb0d38233932e5510de/kolga/libs/helm.py#L123:L127) on line 120, could this be changed into a variable that we could set ourselves?

**Comment by frwickst** [comment]
Hi!

Could you expand a bit on the issues that you are having. The code that you are pointing two uses three variables to set the timeout, all of them configurable by the user.

You have the following ways to configure the startup time:
- `K8S_PROBE_INITIAL_DELAY`: This is the time Kubernetes waits before doing any checks for the application (default: 60)
- `K8S_PROBE_FAILURE_THRESHOLD`: This is the times the health check can fail before the application is seen as unhealthy (default: 3)
- `K8S_PROBE_PERIOD`: This is the time between health checks (default: 10)

Say that you have the default values in use and you are deploying a single instance of an application, that would give you a timeout of:
(1 * 60) + (3 * 10) + 120 = 210 seconds (3,5min)

In other words, any startup time under 3,5min should not cause issues.

Say that you have an application that needs between 8-10min to start up. Then you can do something in the lines of:

```
K8S_PROBE_INITIAL_DELAY=480 (7,5min)
K8S_PROBE_FAILURE_THRESHOLD=6
K8S_PROBE_PERIOD=20
```

This will give you a timeout of:
(1 * 480) + (6 * 20) + 120 = 720 (12 min)

and a start time of the application must be less than 720-120 = 600 (10min)
